### PR TITLE
Send posts to slack

### PR DESCRIPTION
**Pull request by swemail**
Send posts to slack on creating posts

### CODE REVIEW

**Review by transient-jonas — APPROVED** [review]
LGTM 👍

File: functions/src/lib/translation.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      from,
+      to,
+    });
+
+    return translatedMessage;
+  } catch (error) {
+    console.error(error);
+    return message;
+  }
+};
```
Comment (gewfy) [inline functions/src/lib/translation.ts:22]: Nice!!

File: functions/src/models/slack.ts [hunk 2]
```diff
@@ -99,6 +103,77 @@ const createResponseBlocks = (
...
+    {
+      type: 'section',
+      text: {
+        type: 'mrkdwn',
+        text,
+      },
+    },
+  ];
+
+  if (originalText.length >= SHARING_POST_MIN_LENGTH) {
```
Comment (gewfy) [inline functions/src/models/slack.ts:139]: More niceness!

**Review by gewfy — APPROVED** [review]
Wow massive! Great work!

**Comment by gewfy** [comment]
Here's a little re-usable todo when adding env-variables:
- [ ] Add `SLACK_SHARING_POSTS_CHANNEL` to:
  - [ ] `INSTALLATION.md` 
  - [ ] `.env.example`
  - [ ] jest setup
  - [ ] workflows
  - [ ] secrets

**Pull request merged**